docs/how_to_guides/creating_batches/how_to_load_a_database_table_view_or_query_result_as_a_batch.rst

```diff
@@ -0,0 +1,125 @@
+
+
+.. admonition:: Prerequisites -- this how-to guide assumes you have already:
+
+  - :ref:`Set up a working deployment of Great Expectations <getting_started>`
+  - :ref:`Configured a SQL datasource <_how_to_guides__configuring_datasources>`
+  - Identified a ``table`` name, ``view`` name, or ``query`` that you would like to use as the data to validate.
+  - Optionally, configured a Batch Kwargs Generator to support inspecting your database and dynamically building Batch Kwargs.
+
+
+Steps
+-----
+
+#. Construct Batch Kwargs that describe the data you plan to validate.
+
+    For example, if your configured datasource is named "my_db" and you plan to validate data in a view called "my_view" located in the "my_schema" schema:
+
+    - To manually build ``table`` or ``view``-based Batch Kwargs, create a dictionary:
+
+        .. code-block:: python
+
+            batch_kwargs = {
+                "datasource": "my_db",
+                "schema": "my_schema",  # schema is optional; default schema will be used if it is omitted
+                "table": "my_view"  # note that the "table" key is used even to validate a view
+            }
+
+    - To use a Table Batch Kwargs Generator, call the ``build_batch_kwargs`` method:
+
+        .. code-block:: python
+
+            batch_kwargs = context.build_batch_kwargs("my_db", "my_generator", "my_view")
+
+    - To manually build ``query``-based Batch Kwargs, create a dictionary:
+
+        .. code-block:: python
+
+            batch_kwargs = {
+                "datasource": "my_db",
+                "query": "SELECT * FROM my_schema.my_table WHERE '1988-01-01' <= date AND date < '1989-01-01';
+            }
+
+    - To use a Query Batch Kwargs Generator, call the ``build_batch_kwargs`` method:
+
+        .. code-block:: python
+
+            batch_kwargs = context.build_batch_kwargs(
+                "my_db",
+                "queries",
+                "movies_by_date",
+                query_parameters={
+                    "start": "1988-01-01",
+                    "end": "1989-01-01"
+                }
+            )
+
+#. **Obtain an Expectation Suite to use to validate your batch**.
+
+    .. code-block:: python
+
+        expectation_suite_name = "npi.warning"  # choose an appropriate name for your suite
+
+    If you have not already created a suite, you can do so now.
+
+    .. code-block:: python
+
+        # Note, you can add the "overwrite_existing" flag to the below command if the suite
+        # exists but you would like to replace it.
+        context.create_expectation_suite(expectation_suite_name)
+
+
+#. **Get the batch to validate**.
+
+    .. code-block:: python
+
+        batch = context.get_batch(
+            batch_kwargs=batch_kwargs,
+            expectation_suite_name=expectation_suite_name
+        )
+
+#. **Use your batch to create Expectations or validate the data**
```

Comment:
Instead of adding this as its own step, how about just including it as regular text?

"Now that you have a Batch, you can use it to create Expectations or validate the data"

Reply:
👍